src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/DiagnosticsOverlay.xaml

```diff
@@ -449,6 +449,11 @@
 						Click="OnThemeButtonClicked"
 						Style="{StaticResource DiagnosticsOverlayButtonStyle}" />
 
+				<!-- Star rating Button -->
```

Comment:
Should we keep this test in the overlay or move it to the diagnostic page?

Reply:
I like it in the overlay personally putting it in the diagnostic page is just extra steps added when we wanna test imo

Reply:
True, but we still need to keep the overlay clean for developer, it's not something that we will use 100x time.

Reply:
So basically doesn't need to be very accessible I think.

Reply:
@jeanplevesque What do you think